save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/ReactorUtils.kt

```diff
@@ -128,6 +196,22 @@ fun <T> T.toFluxByteBufferAsJson(objectMapper: ObjectMapper): Flux<ByteBuffer> =
  */
 fun <T : Any, K : Comparable<K>> Flux<T>.sortBy(keyExtractor: (T) -> K): Flux<T> = sort(Comparator.comparing(keyExtractor))
 
+/**
+ * Exposes this tuple as a data class, allowing destructuring.
+ *
+ * @return the 1st component of this tuple.
+ */
+operator fun <T : Any> Tuple2<T, *>.component1(): T =
```

Comment:
there is
```kotlin
import reactor.kotlin.core.util.function.component1
import reactor.kotlin.core.util.function.component2
```

why it doesn't fit for us?

Reply:
Was unaware of those, thanks!

Addressed with 9660cb5e85bef5e35aa3c733cd2c47c96893c653.